kerl

```diff
@@ -894,6 +899,17 @@ endif
 
 rehash
 ACTIVATE_CSH
+
+    otpver=$(get_otp_version "$rel")
```

Comment:
If this kernel flag has no effect on OTPs before 20, I'd be inclined to do away with the conditional on `get_otp_version` and just always dump this in `activate` etc. I think it might keep things a bunch neater. What do you reckon @mrallen1 ?